PhpUnit/ContainerBuilderHasServiceDefinitionConstraint.php

```diff
@@ -4,13 +4,15 @@
 
 use PHPUnit\Framework\Constraint\Constraint;
 use PHPUnit\Framework\Constraint\IsEqual;
+use SebastianBergmann\Exporter\Exporter;
```

Comment:
This will break when using the phpunit phar release, see #158.

Reply:
yeah, i just noticed your merge request and was wondering whether it isn't a phar problem when a class is incorrectly vendored?

phpunit 11 does no longer include the exporter so not adding this dependency isn't really an option for this mr

Reply:
> phpunit 11 does no longer include the exporter

Nope, it's definitely there in the [composer.json for v11](https://github.com/sebastianbergmann/phpunit/blob/11.0.0/composer.json), and also in the v11 phar release, and still vendored as `PHPUnit\SebastianBergmann\Exporter\Exporter`.

> was wondering whether it isn't a phar problem when a class is incorrectly vendored

I'm pretty sure it's supposed to be so. The phpunit phar release basically vendor all it's dependencies under the `PhpUnit` root namespace.

Reply:
> not adding this dependency isn't really an option for this pr

You could add a compatibility factory, i.e. something like this:
```php
use PhpUnit\SebastianBergmann\Exporter\Exporter as VendoredExporter;
use SebastianBergmann\Exporter\Exporter;

class PhpUnitCompatFactory {
  public static function exporter(): Exporter|VendoredExporter {
    return class_exists(VendoredExporter::class) ? new VendoredExporter() : new Exporter();
  }
}
```

Reply:
Ah wait, I just noticed that you explicitely require `sebastian/exporter` in this PR, so this will probably work fine as long as the `Exporter` instances stay private.

Reply:
yeah, i assume the phar vendors the exporter differently to prevent conflicts with a different version explicitly being installed in the repository so the actual fix is adding the `sebastian/exporter` to the composer requirements